util/chunk/chunk_util_test.go

```diff
@@ -81,3 +81,36 @@ func BenchmarkAppendSelectedRow(b *testing.B) {
 		}
 	}
 }
+
+func BenchmarkGetInt64InColumn(b *testing.B) {
+	numRows := 1024
+	srcChk := newChunkWithInitCap(numRows, 8)
+	for i := 0; i < numRows; i++ {
+		srcChk.AppendRow(MutRowFromValues(2).ToRow())
+	}
+	b.ReportAllocs()
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		GetInt64InColumn(srcChk, 0)
```

Comment:
here we should also add a `append()` function call to simulate the operation in `IndexLookupReader`.